Dockerfile

```diff
@@ -1,7 +1,10 @@
 # Inherit from an upstream image
-FROM pangeo/pangeo-notebook:2023.11.14
+FROM pangeo/pangeo-notebook:2024.07.30
```

Comment:
This is the plain notebook, you probably want one of the ml specific images? There is a pytorch and tensorflow version (see [here](https://github.com/pangeo-data/pangeo-docker-images#pangeo-docker-images))